src/session-feedback/session-feedback.interface.ts

```diff
@@ -0,0 +1,10 @@
+import { ISession } from 'src/session/session.interface';
```

Comment:
Could I be a pain and say lets get rid of this interface and just use the Entity?

Reply:
Just to clarify, you mean I should completely remove the session-feedback.interface? I think I am only using the SessionFeedbackEntity in the session-feedback.service anyway so this should not be a problem. Thanks!

Reply:
Yes please!

Reply:
Removed now :)